client/state/selected-editor/reducer.js

```diff
@@ -7,10 +7,10 @@
 /**
  * Internal dependencies
  */
-import { EDITOR_TYPE_SET } from 'state/action-types';
+import { EDITOR_TYPE_RECEIVE } from 'state/action-types';
 import { keyedReducer } from 'state/utils';
 
 export const selectedEditor = ( state, { type, editor } ) =>
-	type === EDITOR_TYPE_SET ? editor : state;
+	type === EDITOR_TYPE_RECEIVE ? editor : state;
```

Comment:
for similar reasons to `_FETCH` we have avoided `_RECEIVE` due to the way it couples the state handling with network activity. we want to be able to interact with state from the dev console in a way that doesn't seem like it's mashing two unrelated systems: network synchronization of data and app state.

to that end I'd personally prefer to say `dispatch( { type: EDITOR_TYPE_SET, editor: 'gutenberg-iframe' } )` than to use `EDITOR_TYPE_RECEIVE` because I don't know what `_RECEIVE` means in context of the dev console